### PR TITLE
Functional create task API

### PR DESCRIPTION
**Pull request by thilan-tran**
- postTask API integrated, can now add new Tasks to backend
- all functional buttons on tasklisk (select tasks, complete tasks, schedule tasks) have working logic
    - now just have to integrate backend API with that logic
- UI updates for login page

Creating tasks and performing operations:

Creating_Tasks

### CODE REVIEW

File: src/components/containers/Login.js
Comment (alexlongerbeam) [inline src/components/containers/Login.js]: What is this handleUser?

File: src/components/containers/TaskList.js
Comment (alexlongerbeam) [inline src/components/containers/TaskList.js]: Why are we filling these out? The task returned should have this info (as of a recent backend change). Can we default to what the returned task says?

File: src/components/containers/TaskList.js
Comment (alexlongerbeam) [inline src/components/containers/TaskList.js]: A new commit by Alvin actually created the set task complete function in the Task Service, rebase and you should have it

**Review by alexlongerbeam — CHANGES_REQUESTED** [review]
Lets do a bit more work on this before merging. Let's get create task fully hooked up with the new backend model. (I also tried to build this branch locally and it errored out).

Things to do for this branch:
1. Use the new postTask function and actually add the user-specified due date. (use `Date.toISOString()` to format the due date before sending
2. Use the backend-defined model that is returned after creating a task, and don't fill in any fields
3. Set the task complete using Alvin's new function

I merged Alvin's and I'm going to merge your UI PR, so make sure to rebase this branch on top of the latest master before making any changes, there will most likely be merge conflicts. Happy to walk through the rebase process with you at meeting if you're rusty

File: src/components/containers/TaskList.js
Comment (alexlongerbeam) [reply src/components/containers/TaskList.js]: Also, please send the user-specified due date to the backend when creating

File: src/components/containers/Login.js
Comment (thilan-tran) [reply src/components/containers/Login.js]: Leftover code from working on a dynamic navbar. Has been removed in latest commit.

File: src/components/containers/TaskList.js
Comment (thilan-tran) [reply src/components/containers/TaskList.js]: Updated to match database models.

File: src/components/containers/TaskList.js
Comment (thilan-tran) [reply src/components/containers/TaskList.js]: Updated to use complete task API.

**Review by alexlongerbeam — APPROVED** [review]
Overall it's good so accepting to unblock **BUT**

If I added multiple tasks and scheduled them all, then I complete 1, they all disappear.

Once you fix that, feel free to merge this in!

**Pull request merged**
